### PR TITLE
Hooks

### PR DESCRIPTION
**Pull request by live627**
Some hooks I'd like to see in the core.

### CODE REVIEW

File: Themes/default/languages/Errors.english.php [hunk 1]
```diff
@@ -346,7 +346,6 @@
 $txt['loadavg_unreadreplies_disabled'] = 'The server is currently under high stress.  Please try again shortly.';
 $txt['loadavg_show_posts_disabled'] = 'Please try again later.  This member\'s posts are not currently available due to high load on the server.';
 $txt['loadavg_unread_disabled'] = 'The server\'s resources are temporarily under too high a demand to list out the topics you have not read.';
-$txt['loadavg_userstats_disabled'] = 'Please try again later.  This member\'s statistics are not currently available due to high load on the server.';
```
Comment (emanuele45) [inline Themes/default/languages/Errors.english.php:349]: Why did you remove this string?
This is used.

Comment (live627) [inline Themes/default/languages/Errors.english.php:349]: Oh crap, how'd that get there? Whatever, I'll get that back.
